### ISSUE TITLE
Optional: Dockerize the project🐋

### ISSUE THREAD

**Issue by deepanshu-rawat6** [upes-open/OSoC-bio.lnk#16]

**Comment by ChrisJon02** [comment]
I want to work on this issue. But when should I actually work on this? ...since dockerizing is one of the last steps before deploying

**Comment by deepanshu-rawat6** [comment]
Yes @ChrisJon02 , I'll assign you this issue, but you'll have to work on this, whenever a service get completed

**Comment by ChrisJon02** [comment]
Well, I'll do that @deepanshu-rawat6 .
Thank you

**Comment by Siddharthaaa21** [comment]
could work on that !

**Comment by deepanshu-rawat6** [comment]
@Siddharthaaa21 This issue is already assigned please check other issues. Best of luck✨
